src/routes/view-content/[guideId]/[bibleSection]/+page.svelte

```diff
@@ -0,0 +1,418 @@
+    import { _ as translate } from 'svelte-i18n';
+    import CompassIcon from '$lib/icons/CompassIcon.svelte';
+    import NavMenuTabItem from './NavMenuTabItem.svelte';
+    import ButtonCarousel from '$lib/components/ButtonCarousel.svelte';
+    import TopNavBar from './TopNavBar.svelte';
+    import ErrorMessage from '$lib/components/ErrorMessage.svelte';
+    import {
+        createMultiClipAudioState,
+        type MultiClipAudioState,
+    } from '$lib/components/AudioPlayer/audio-player-state';
+    import { objectKeys } from '$lib/utils/typesafe-standard-lib';
+    import {
+        type BibleData,
+        type ResourceData,
+        fetchBibleContent,
+        fetchResourceData,
+        fetchBibleData,
+    } from './data-fetchers';
+    import { preferredBibleIds } from '$lib/stores/preferred-bibles.store';
+    import type { BibleBookContentDetails } from '$lib/types/bible';
+    import {
+        cacheBibleMetadata,
+        cacheBiblesForBibleSection,
+        getBibleBookCodesToName,
+    } from '$lib/utils/data-handlers/bible';
+    import { isOnline } from '$lib/stores/is-online.store';
+    import { currentLanguageInfo } from '$lib/stores/language.store';
+    import MainMenu from '$lib/components/MainMenu.svelte';
+    import { guideResources } from '$lib/stores/parent-resource.store';
+    import {
+        PassagePageMenuEnum,
+        passagePageShownMenu,
+        openGuideMenu,
+        recalculatePanesAndMenus,
+    } from '$lib/stores/passage-page.store';
+    import GuideMenu from './guide-menu/GuideMenu.svelte';
+    import { onMount } from 'svelte';
+    import LibraryResourceMenu from './library-resource-menu/LibraryResourceMenu.svelte';
+    import BibleMenu from './bible-menu/BibleMenu.svelte';
+    import BookPassageSelectorPane from './bible-menu/BookPassageSelectorPane.svelte';
+    import BookChapterSelectorPane from './bible-menu/BookChapterSelectorPane.svelte';
+    import {
+        MediaType,
+        type ResourceContentInfoWithMetadata,
+        type TextResourceContentJustId,
+    } from '$lib/types/resource';
+    import SettingsMenu from './settings-menu/SettingsMenu.svelte';
+    import type { BibleSection } from '$lib/types/bible';
+    import QuickShare from './quick-share/QuickShare.svelte';
+    import GuideContent from './guide-content/GuideContent.svelte';
+    import FullscreenTextResource from './library-resource-menu/FullscreenTextResource.svelte';
+    import type { Language } from '$lib/types/file-manager';
+    import { filterBoolean } from '$lib/utils/array';
+    import Feedback from './feedback/Feedback.svelte';
+    import BibleViewer from './BibleViewer.svelte';
+    import { page } from '$app/stores';
+    import { ContentTabEnum, createContentContext } from './context';
+
+    const {
+        currentGuide,
+        currentBibleSection,
+        currentTab,
+        syncSelectedGuideFromUrlParam,
+        syncSelectedBibleSectionFromUrlParam,
+    } = createContentContext();
```

Comment:
The context is created here at the root level of the page and can be used by any of the page's descendants.